reference/library/src/webapp/js/headscripts.js

```diff
@@ -496,7 +496,7 @@ function showNotif(item, button,formName)
 }
 
 // stuff to do auto-update using the XMLHttpRequest object
-var updateReq;
+var updateReq = null;
```

Comment:
Isn't this the only bit of the change that is actually needed. I can't understand why any other changes are needed.